### PR TITLE
fix(css): Prevent lowercasing css modules variables within media queries

### LINKED ISSUE #6468: Prevent lowercasing css modules variables within media queries
Is it possible to disable lowercasing for css modules variables within the media query?
Input: 
```
@value bpMedium, bpLarge from '@styles/breakpoints.module.css';

.container {
  margin-right: auto;
  margin-left: auto;
  width: 100%;
  padding-left: 20px;
  padding-right: 20px;
}

@media (bpMedium) {
  .container {
    width: 768px;
  }
}

@media (bpLarge) {
  .container {
    width: 960px;
  }
}

```

Transforms into 
```
@value bpMedium, bpLarge from '@styles/breakpoints.module.css';

.container {
  margin-right: auto;
  margin-left: auto;
  width: 100%;
  padding-left: 20px;
  padding-right: 20px;
}

@media (bpmedium) {
  .container {
    width: 768px;
  }
}


@media (bplarge) {
  .container {
    width: 960px;
  }
}
```
It's related with that closed issue https://github.com/prettier/prettier/issues/4110 however the pull request solved only lowercasing in property values.

### PR DESCRIPTION
**Pull request by pranshuchittora**
Fixes #6468 

TODO
- [ ] Tests

-----
**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by pranshuchittora** [comment]
After this function gets called, the node value changes to lower case.
https://github.com/prettier/prettier/blob/8d0ac8bb58ccc0d30a47a000ca3adba7343cec90/src/main/core.js#L86

**Comment by pranshuchittora** [comment]
https://github.com/prettier/prettier/blob/8d0ac8bb58ccc0d30a47a000ca3adba7343cec90/tests/css_case/case.css#L7

The above test states that all UPPERCASE should be parsed to lowercase.
Hence, I am not sure about proceeding further. :confused:

**Comment by pranshuchittora** [comment]
What about this fix??
```diff

diff --git a/src/language-css/printer-postcss.js b/src/language-css/printer-postcss.js
index dc31d53..525a5b6 100644
--- a/src/language-css/printer-postcss.js
+++ b/src/language-css/printer-postcss.js
@@ -279,9 +279,7 @@ function genericPrint(path, options, print) {
       return concat(["(", concat(path.map(print, "nodes")), ")"]);
     }
     case "media-feature": {
-      return maybeToLowerCase(
-        adjustStrings(node.value.replace(/ +/g, " "), options)
-      );
+      return node.value;
     }
     case "media-colon": {
       return concat([node.value, " "]);
(END)


```

File: src/language-css/utils.js [hunk 1]
```diff
@@ -377,9 +377,7 @@ function isColonNode(node) {
 }
 
 function isMediaAndSupportsKeywords(node) {
-  return (
-    node.value && ["not", "and", "or"].indexOf(node.value.toLowerCase()) !== -1
-  );
+  return node.value && ["not", "and", "or"].indexOf(node.value) !== -1;
```
Comment (jamesgeorge007) [inline src/language-css/utils.js]: Maybe use `includes()` instead .

Comment (evilebottnawi) [reply src/language-css/utils.js]: No need

**Pull request closed without merging**